### DISCUSSION TITLE
AutoGluonTabularModel with dynamic features

### DISCUSSION THREAD

**Discussion by fsty** [autogluon/autogluon#2508] [Q&A]
Hi !
How to enable time-varying covariates in AutoGluonTabular for  time series ?

**Comment by shchur** [comment]
Hi, this is currently not supported but we plan to include it in the next major release.

**Reply by fsty** [reply]
Thank you @shchur it will be very cool, maybe you can answer another question about TFT, this model support unkown time varing features : here is an example: https://pytorch-forecasting.readthedocs.io/en/stable/tutorials/stallion.html 
do you plan to support this kind of features  ?

**Reply by shchur** [reply]
~~I guess it's related to #2489. This is currently not on the roadmap~~

**Reply by shchur** [reply]
Sorry, it seems that I misunderstood your question. We plan to add support for unknown time-varying covariates sometime in the future.
